boards/nxp/frdm_mcxn947/board.c

```diff
@@ -146,6 +146,9 @@ static int frdm_mcxn947_init(void)
 	CLOCK_SetClkDiv(kCLOCK_DivPLL1Clk0, 1U);
 #endif
 
+	/* Enable 1MHz clock. */
+	SYSCON->CLOCK_CTRL |= SYSCON_CLOCK_CTRL_FRO1MHZ_CLK_ENA_MASK;
```

Comment:
Why do we need this clock?

Reply:
Yes, this is needed. I could move this to the I3C specific code. However I suspect there are other peripherals that would also need this clock. Hence added it here.

Reply:
I think we should keep this conditional under the I3C specific code until multiple consumers exist

Reply:
Updated